server/app/_components/TodoEditor.tsx

```diff
@@ -59,11 +73,41 @@ const TodoEditor = ({ editTargetTodo, onSubmit }): JSX.Element => {
           <button
             type="submit"
             className="flex w-full justify-center rounded-md bg-indigo-600 px-3 py-1.5 text-sm font-semibold leading-6 text-white shadow-sm hover:bg-indigo-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600"
-            onClick={() => onSubmit(todo)}
+            onClick={() => openDialog("確認画面","ToDoリストに追加しますか？")}
           >
             保存
           </button>
         </div>
+
+        
+        <div>
+          <dialog 
+            id="myDialog"
```

Comment:
`id`要素を置く代わりにReact.useRefを置いてやれば、コード内で簡単に要素を特定できます。
```suggestion
            ref={dialogRef}
```